salt/modules/lxc.py

```diff
@@ -989,6 +1045,10 @@ def init(name,
     if users is None:
         users = []
     dusers = ['root']
+    # think to touch the default user generated by default ubuntu template which
+    # has a really unsecure password too ...
+    if __grains__['os'] in ['Ubuntu'] and 'ubuntu' not in users:
+        dusers.append('ubuntu')
```

Comment:
The problem is that you hold too much (6monthes....) the module while i had to commit stuff (stuff as features but also bugfix of existing tree and your introduced refactor bits).
I let you remeber that you let met no choice than forking and you ever refused in clever ways to merge our concurrent work. So what's needed to be done in time could not be done.
Now my module differs a lot, so we need to reconcile back somewhere the module as a whole.

It can not more be splitten in clean separate commits.

And it can not more, also, for me considered as a bugfix commit as it is a reconcibile back work which shall be stabilized before going back into release branch.

cc @thatch45